src/adapter.js

```diff
@@ -192,10 +317,20 @@ export function parse({minim, source}, done) {
         }
 
         // Currently, default responses are not supported in API Description format
-        const relevantResponses = _.omit(methodValue.responses, 'default');
+        // TODO: [Annotation] Add warning about not showing the default!
+        const relevantResponses = _.chain(methodValue.responses)
+          .omit('default')
+          .omit(isExtension)
+          .value();
 
         if (_.keys(relevantResponses).length === 0) {
-          createTransaction(minim, transition, method);
+          if (bodyParameters.length) {
+            // Create an empty successful response so that the request/response
+            // pair gets properly generated.
+            relevantResponses[200] = {};
```

Comment:
@danielgtaylor @netmilk So we include an empty response rather than no response or one with status in it. Responses have no required attributes in the Refract namespace, but a response element is a MUST. Therefore, to satisfy namespace, empty response makes sense to me.